src/lib/journey/stages/generic.svelte

```diff
@@ -118,8 +120,10 @@
   {/each}
 
   {#if !stepMetadata.isStepSelfSubmittable}
-    <Button busy={loading} style="primary" type="submit" width="full">
+    <Button busy={journey?.loading} style="primary" type="submit" width="full">
       <T key="nextButton" />
     </Button>
   {/if}
+
+  <BackTo {journey} />
```

Comment:
This is where the "back to" button is rendered when the user jumps from one journey to another.